scenarios/turbine-heat-processor/README.md

```diff
@@ -0,0 +1,41 @@
+# Turbine Heat Processor
+A Flink streaming application for processing temperature data from a Pravega `Stream`. 
+Complements the Turbine Heat Sensor app 
+([`turbineheatsensor`](https://github.com/pravega/pravega-samples/scenarios/turbine-heat-sensor)). 
+The application computes a daily summary of the temperature range observed on that day by each sensor.
+
+Automatically creates a scope (default: `examples`) and stream (default: `turbineHeatTest`) as necessary.
+
+## Pre-requisites
+1. Pravega running (see [here](http://pravega.io/docs/latest/getting-started/) for instructions)
+2. Build [pravega-samples](https://github.com/pravega/pravega-samples) repository
+3. Apache Flink running
+
+
+## Execution
+Run the sample from the command-line:
+```
+$ bin/run-example [--controller <URI>] [--input <scope>/<stream>] [--startTime <long>] [--output <path>]
+```
+
+Alternately, run the sample from the Flink UI.
+- JAR: `pravega-flink-examples-0.1.0-SNAPSHOT-all.jar`
```

Comment:
Can we rename the file name to `pravega-flink-examples-<VERSION>-all.jar`?

Reply:
Changed.